gomod-check/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "go.mod check"
+description: "Checks that the go.mod doesn't contain replacements for 'protected' modules. Assumes go, jq and grep are available."
```

Comment:
curious, do you know what happens when they are not available?

Reply:
The action passes when there are no protected modules: https://github.com/metlos/gh-action-test/actions/runs/8344584416

Reply:
and it also passes (or actually doesn't even run) if there is no `go.mod` in the repo: https://github.com/metlos/gh-action-test/actions/runs/8344627115